### DISCUSSION TITLE
block user

### DISCUSSION THREAD

**Discussion by mohsenl99** [Docile-Alligator/Infinity-For-Reddit#1208] [Completed Ideas]
I can't find an option to block user in the app.
please add it

**Comment by Docile-Alligator** [comment]
It was removed. Reddit messed up Infinity's access token and it returns 403 error when blocking a user.

**Reply by mohsenl99** [reply]
Oh...
it means you never add it even in future??

**Reply by Wladefant** [reply]
we will probably implement it again, when reddit fixes the api
